aspnetcore/mvc/views/overview.md

```diff
@@ -1,228 +1,336 @@
-When this action returns, the *About.cshtml* view shown above is rendered:
+![About page rendered in the Edge browser](overview/_static/about-page.png)
 
-![About page](overview/_static/about-page.png)
+The `View` helper method has several overloads. You can optionally specify:
 
-### View Discovery
+* An explicit view to return:
 
-When an action returns a view, a process called *view discovery* takes place. This process determines which view file will be used. Unless a specific view file is specified, the runtime looks for a controller-specific view first, then looks for matching view name in the *Shared* folder.
+  ```csharp
+  return View("Orders");
+  ```
+* A [model](xref:mvc/models/model-binding) to pass to the the view:
 
-When an action returns the `View` method, like so `return View();`, the action name is used as the view name. For example, if this were called from an action method named "Index", it would be equivalent to passing in a view name of "Index". A view name can be explicitly passed to the method (`return View("SomeView");`). In both of these cases, view discovery searches for a matching view file in:
+  ```csharp
+  return View(Orders);
+  ```
+* Both a view and a model:
 
-   1. Views/\<ControllerName>/\<ViewName>.cshtml
+  ```csharp
+  return View("Orders", Orders);
+  ```
 
-   2. Views/Shared/\<ViewName>.cshtml
+### View discovery
 
->[!TIP]
-> We recommend following the convention of simply returning `View()` from actions when possible, as it results in more flexible, easier to refactor code.
+When an action returns a view, a process called *view discovery* takes place. This process determines which view file is used based on the view name. 
 
-A view file path can be provided instead of a view name. If using an absolute path starting at the application root (optionally starting with "/" or "~/"), the *.cshtml* extension must be specified as part of the file path (for example, `return View("Views/Home/About.cshtml");`). Alternatively, you can use a relative path from the controller-specific directory within the *Views* directory to specify views in different directories (for example, `return View("../Manage/Index");` inside the `HomeController`). Similarly, you can traverse the current controller-specific directory (for example, `return View("./About");`). Note that relative paths don't use the *.cshtml* extension. As previously mentioned, follow the best practice of organizing the file structure for views to reflect the relationships among controllers, actions, and views for maintainability and clarity.
+When an action returns the `View` method (`return View();`) and a view isn't specified, the action name is used as the view name. For example, the *About* `ActionResult` method name of the controller is used to search for a view file named *About.cshtml*. First, the runtime looks in the *Views/[ControllerName]* folder for the view. If it doesn't find a matching view there, it searches the *Shared* folder for the view.
```

Comment:
"When an action returns the View method (return View();) and a view isn't specified, the action name is used as the view name." > "The default behavior of the View method (return View();) is to return a view with the same name as the action method from which it is called."